core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1363,6 +1363,50 @@ public Set<String> getNamespaceNames() throws ExecutionException {
     }
   }
 
+  @Override
+  public void upgrade(Map<String, String> options) throws ExecutionException {
+    if (!metadataTableExists()) {
+      return;
```

Comment:
Similar question to the above one

Reply:
My answer is the same as https://github.com/scalar-labs/scalardb/pull/1204#discussion_r1364843384